docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -41,13 +41,18 @@
    "metadata": {},
```

Comment:
<p>I'm thinking this maybe too much for a getting started tutorial, would remove this for now and refer again to the docs of <code>default_progress_fn</code> for example of a more complex callback.</p><p>Later, I'd add a new notebook on how to use progress bars when it's implemented for low-rank Sinkhorn and/or GW + add a link to the "Going further" section.</p><p>Pinging @marcocuturi @pierreablin for opinion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Sure, as we discussed offline, I have split the progress bar tutorial in its own notebook and reverted changes to the `basic_ot_between_datasets.ipynb` notebook.